compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1257,8 +1259,11 @@ void buildLLVMGPUCodegenPassPipeline(OpPassManager &variantPassManager,
   {
     OpPassManager &modulePassManager = variantPassManager.nest<ModuleOp>();
     modulePassManager.addPass(createLowerExecutableUsingTransformDialectPass());
+    LLVMGPULowerExecutableTargetPassOptions options;
+    options.forROCDL = useROCM;
     FunctionLikeNest(modulePassManager)
-        .addPass(createLLVMGPULowerExecutableTargetPass)
+        .addPass(
+            [&]() { return createLLVMGPULowerExecutableTargetPass(options); })
```

Comment:
nit: empty parens can be omitted in lambda expressions
```suggestion
            [&] { return createLLVMGPULowerExecutableTargetPass(options); })
```